src/evaluate.jl

```diff
@@ -137,6 +137,15 @@ function evaluate!{T<:NumberNotSeriesN}(x::Array{TaylorN{T},1}, δx::Array{Taylo
     nothing
 end
 
+function evaluate!{T<:NumberNotSeriesN}(x::Array{TaylorN{T},1}, δx::Array{TaylorN{T},1},
+        x0::Array{TaylorN{T},1})
+    @assert length(x) == length(x0)
+    @inbounds for i in eachindex(x)
+        x0[i] = evaluate( x[i], δx )
```

Comment:
Perhaps using here (as well in similar occurrences in other methods) `evaluate!(x[i], δx, x0[i])` instead, yields better performance.

Reply:
Agreed for `evaluate` methods but not for `evaluate!` methods as `evaluate!`, as the docs say, just work for arrays.

Reply:
Good point!